src/platforms/mod.rs

```diff
@@ -0,0 +1,43 @@
+use bevy::prelude::*;
+use bevy_rapier2d::prelude::*;
+pub struct PlatformsPlugin;
+
+impl Plugin for PlatformsPlugin {
+    fn build(&self, app: &mut App) {
+        app.add_startup_system(spawn_ground.after("setup_physics"));
+    }
+}
+
+/// Spawn ground under player
+fn spawn_ground(
+    mut commands: Commands,
+    rapier_config: Res<RapierConfiguration>,
+    asset_server: Res<AssetServer>,
+) {
+    let collider_size_hx = 2000.0 / rapier_config.scale / 2.0;
+    let collider_size_hy = 24.0 / rapier_config.scale / 2.0;
```

Comment:
This seems to be a fixed size.  Should we eventually make it so this extends off screen?